roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
I agree that going with a hierarchy here is the wrong approach... 

That said, I'm not sure that I'm a fan of the mixin approach here either. Maybe we should go with decorators?

On a more serious note, I wonder if it would make sense to do something crazy with metaclasses, where we can register test "plugins" that match the host profile and then execute them as needed. It does add a bit of "magic", but might make the individual test plugins easier to write and maintain.